rsts/deployment/cluster_config/customizable_resources.rst

```diff
@@ -5,8 +5,10 @@ Adding New Customizable Resources
 ##################################
 
 For background on customizable resources, see the :ref:`deployment-cluster-config-general`.
```

Comment:
Can we have this as a note after "Examples of such resources include execution clusters, task resource defaults, and more."?